string robot_id
geometry_msgs/PoseStamped plug_pose
---
bool success
string message